.vib/spring-cloud-dataflow/cypress/cypress/integration/spring-cloud-dataflow_spec.js

```diff
@@ -0,0 +1,108 @@
+  cy.contains('button', 'Create stream(s)').click();
+  cy.get('.modal-content').should('contain', 'Create Stream');
+  cy.fixture('streams').then((stream) => {
+    cy.get('input[placeholder="Stream Name"]').type(
+      `${stream.newStream.name}-${random}`
+    );
+    cy.contains('.btn-primary', 'Create the stream').click();
+    cy.contains(
+      '.datagrid-inner-wrapper',
+      `${stream.newStream.name}-${random}`
+    );
+  });
+  cy.contains('.toast-container', 'successfully');
+  cy.contains('clr-dg-cell', 'UNDEPLOYED')
+    .siblings('clr-dg-cell', 'test-stream')
+    .first()
+    .click();
+  cy.contains('button#btn-deploy', 'Deploy stream').click();
+  cy.contains('button', 'Deploy stream').click();
+  cy.get('.toast-container').should('contain', 'Deploy success');
+});
+
+it('allows a task to be scheduled and destroyed', () => {
+  const CRON_EXPRESSION = '*/5 * * * *';
+  const TASK_APPLICATION = 'Task application starters for Maven';
+  const TASK_TYPE = 'timestamp';
+
+  importAnApplication(TASK_APPLICATION);
+  createATask(TASK_TYPE);
+  cy.get('[routerlink="tasks-jobs/tasks"]').click();
+  cy.contains('clr-dg-cell', 'UNKNOWN')
+    .siblings('clr-dg-cell', 'test-task')
+    .first()
+    .click();
+  cy.contains('button', 'Schedule').click();
+  cy.fixture('schedules').then((schedule) => {
+    cy.get('input[name="example"]')
+      .first()
+      .type(`${schedule.newSchedule.name}-${random}`);
+  });
+  cy.get('input[name="example"]').last().type(CRON_EXPRESSION);
+  cy.contains('button', 'Create schedule(s)').click();
+  cy.contains('.toast-container', 'Successfully');
+  cy.contains('a', 'Tasks').click();
+  cy.contains('clr-dg-cell', 'UNKNOWN')
+    .siblings('clr-dg-cell', 'test-task-')
+    .first()
+    .click();
+  cy.contains('button', 'Destroy task').click();
+  cy.contains('.btn-danger', 'Destroy the task').click();
+  cy.contains('.toast-container', '1 task definition(s) destroyed.');
+});
+
+it('allows importing a task from a file and destroying it ', () => {
+  cy.visit('/dashboard');
+  cy.get('[routerlink="manage/tools"]').click();
+  cy.contains('a', 'Import tasks from a JSON file').click();
+  cy.get('input[type="file"]').selectFile(
+    'cypress/fixtures/task-to-import.json',
+    { force: true }
+  );
+  cy.contains('button', 'Import').click();
+  cy.contains('.modal-body', '1 task(s) created');
+  cy.get('.close').click();
+  cy.get('[routerlink="tasks-jobs/tasks"]').click();
+  cy.contains('button', 'Group Actions').click();
+  cy.get('[aria-label="Select All"]').click({ force: true });
+  cy.contains('button', 'Destroy task').click();
+  cy.contains('.modal-content', 'Confirm Destroy Task');
+  cy.contains('button', 'Destroy the task').click();
+  cy.get('.toast-container').should('contain', 'destroyed');
+});
+
+it('allows unregistering of an application', () => {
+  const STREAM_APPLICATION = 'Stream application starters for Kafka/Maven';
```

Comment:
I am guessing this is kind of the `type` of stream application we want to create. If I am right, then I also think this is a characteristic of the application and should be in fixtures.

Reply:
Yes, now I created a fixture for this.
I've been indecisive since we are not exactly creating this application but selecting it from a list.